app/queries/my_facilities/missed_visits_query.rb

```diff
@@ -63,16 +33,34 @@ def calls_made
       .group('facilities.id::uuid')
   end
 
+  def bp_query_by_cohort
```

Comment:
Neat. This generates a hash of BP queries, one for each time period, right?

Reply:
That's correct, I've tried to minimize the changes here so I can return more or less the same data structures from the query. This is so controller and view code stays _mostly_ the same.